src/rendering/layout/ScoreLayout.ts

```diff
@@ -209,23 +208,24 @@ export abstract class ScoreLayout {
         if (notation.isNotationElementVisible(NotationElement.ChordDiagrams)) {
             this.chordDiagrams = new ChordDiagramContainerGlyph(0, 0);
             this.chordDiagrams.renderer = fakeBarRenderer;
-            let chords: Map<string, Chord> = new Map<string, Chord>();
+            let chordNames: Set<string> = new Set<string>();
```

Comment:
The name is not a good indicator for uniqueness. The same chord (by name) might be played differently. That's why in https://github.com/CoderLine/alphaTab/issues/1209 I indicated that the name might be a good first check but we then need to check further based on the details within the chord. 

This might be achieved by pre-computing a unique id in the Chord class based on the information (and we use it here) or we do the check here once and only if needed.